cygnus-ngsi/spec/SPECS/cygnus-ngsi.spec

```
@@ -99,6 +99,11 @@ rm -rf $RPM_BUILD_ROOT
 %{_project_install_dir}
 
 %changelog
+* Thu May 24 2023 Alvaro Vega <alvaro.vegagarcia@telefonica.com> 3.1.0
+- Add: delete entity in lastData when alterationType entityDelete is received (#2231). Requires Orion 3.9+
+- Add: exclude of attribute processing alterationType send by a subscription in all of their values (#2231)
+- Fix: missing blank space in create table if not exists for postgresql
```

```suggestion
- [cygnus-ngsi] Add: delete entity in lastData when alterationType entityDelete is received (#2231). Requires Orion 3.9+
- [cygnus-ngsi] Add: exclude of attribute processing alterationType send by a subscription in all of their values (#2231 
```

Note last line has been removed. As far as I understand from PR https://github.com/telefonicaid/fiware-cygnus/pull/2239, it only applies to cygnus-commons.